toolkit/scripts/tools.mk

```diff
@@ -214,3 +214,9 @@ $(macro_rpmrc): $(toolchain_rpms)
 clean: clean-macro-tools
 clean-macro-tools:
 	rm -rf $(MACRO_DIR)
+
+######## HOST TOOLING ########
+# This section contains a check for the host tooling required by the toolkit.
+__DUMMY_VAR := $(foreach exec,awk curl docker genisoimage git go setfacl tar wget, \
```

Comment:
Do we actually need the dummy var here? Curious what its doing.

Reply:
Thanks for pointing this out. During my tests I had issues to make this `foreach` always run (the `:=` assignment forced the expression to always be evaluated) but I can't repo that issue anymore, so I'll remove the assignment. Maybe something else was just wrong while I was testing this.